lib/ASTGen/Sources/ASTGen/Bridge.swift

```diff
@@ -56,6 +44,30 @@ extension String {
   }
 }
 
+extension SyntaxProtocol {
+  /// Obtains the bridged start location of the node in the source buffer provided by `astgen`.
+  ///
+  /// - Parameter astgen: The visitor providing the `ASTContext` and source buffer.
+  @inline(__always)
+  func bridgedSourceLoc(in astgen: ASTGenVisitor) -> BridgedSourceLoc { 
+    return BridgedSourceLoc(at: self.positionAfterSkippingLeadingTrivia, in: astgen.base)
+  }
+}
+
+extension Optional where Wrapped: SyntaxProtocol { 
```

Comment:
You can shorten this to

```suggestion
extension Optional<SyntaxProtocol> { 
```

Reply:
Thanks for the review! I changed it to `Optional<SyntaxProtocol>`

Reply:
`extension Optional<SyntaxProtocol>` is sugar for `extension Optional where Wrapped == SyntaxProtocol`, not `Wrapped: SyntaxProtocol`.

Reply:
Thanks guys, I've reverted it back to `where Wrapped: SyntaxProtocol`